webpack.config.js

```diff
@@ -141,6 +142,7 @@ module.exports = {
     hot: true,
     port: 9443,
     https: true,
+    host: '0.0.0.0',
```

Comment:
i assume this will make the dev server be accessible externally? do we need this

Reply:
i think Zac added this to be able to run portal rom a dockerfile. i'll remove it